### ISSUE TITLE
Add filter selection bubbles to filtered roster

### ISSUE THREAD

**Issue by JordanGuinn** [ctoec/data-collection#1476]
## Background
Once all of our actual filters have been implemented, we should add more visually distinct indicators to the roster view to inform the user of which filters and filter values they've selected at any given moment.  [Carbon's Tag component](https://react.carbondesignsystem.com/?path=/story/components-tag--default) would be perfect for this kinda thing.

## Acceptance Criteria
- Filter values appear beneath the row of filters when as a user selects them
   - Said values are displayed using [Carbon's Tag component](https://react.carbondesignsystem.com/?path=/story/components-tag--default)
- **All filter values** selected at any given moment are displayed at once
- Filter values disappear when they are deselected from the corresponding filter

## Designs
[Figma View](https://www.figma.com/file/GcYwZ05CjnG6QZNYlsCS1X/ECE-Reporter-%E2%80%93-Final-Designs?node-id=86%3A348)
![Roster - Filter View.PNG](https://images.zenhubusercontent.com/602d78337403e44d7db5c4f3/d7199e20-067d-450a-95c9-1113c914a304)

**Comment by lukenewman** [comment]
@JordanGuinn i don't think this story needs to be blocked by the two remaining filters, right?

**Comment by kmariepat-skylight** [comment]
Notes from grooming: previously deprioritized because implementation was pain, current thought is this is nice to have
